### DISCUSSION TITLE
[Feature request] "Column Type" setting for Number fields

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#10033] [Ideas] [answered]
It'd be great if Number fields had a "Column Type" setting similar to PlainText fields, with options for INT (default), BIGINT etc.

**Comment by brandonkelly** [comment, marked as answer]
Number fields are already smart about which column type they should use, based on the Min, Max, and Decimals settings. Is there a reason you need explicit control?

**Reply by mmikkel** [reply]
Aha! No, I just didn't know that those settings would affect the column type. Putting in the proper max value solves the use case. Thanks Brandon.
